specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/GeneratedSystemEvents.json

```diff
@@ -1349,18 +1352,19 @@
           "$ref": "#/definitions/AcsRouterCommunicationError",
           "description": "Router Communication Inner Error"
         },
-        "details": {
+        "errors": {
           "type": "array",
           "description": "List of Router Communication Errors",
           "items": {
             "$ref": "#/definitions/AcsRouterCommunicationError"
           },
+          "readOnly": true,
           "x-ms-identifiers": []
         }
       },
       "required": [
         "innererror",
-        "details"
```

Comment:
why is this changing names

Reply:
Maybe we are applying a rename and the new decorator is somehow preventing the rename from working.

Reply:
Doesn't look like this is being renamed in propertyNameOverride. But the currently generated name is correct. I wonder if the old version was manually modified.